ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java

```diff
@@ -241,16 +243,18 @@ public void checkButtonStatus(String buttonTitle, String status) {
 
     public Condition conditionValueOf(String status) {
         Condition condition;
-        switch (status) {
-            case "Active":
-            case "Visible":
+        switch (status.toLowerCase()) {
```

Comment:
So yeah, that didn't work:

`And check "Done" button is "enabled"`
![image](https://user-images.githubusercontent.com/9480152/61220930-ad97bd80-a717-11e9-9382-5360889942ee.png)

Reply:
Seems that the "enabled" state is only used in api-provider tests (my bad!), please change all of those to "visible" and this should be good to go